pkg/console/subresource/configmap/brand_okd.go

```diff
@@ -4,5 +4,5 @@ package configmap
 
 const (
 	DEFAULT_BRAND   = "okd"
-	DEFAULT_DOC_URL = "https://docs.okd.io/4.5/"
+	DEFAULT_DOC_URL = "https://docs.okd.io/4.6/"
```

Comment:
Not new, but this should be https://docs.okd.io/latest/

Reply:
@spadgett but then the 4.6 release will always point to the latest, instead to the particular version. So in case we will be working on 4.7 then the 4.6 will be pointing to latest.

Reply:
OKD *only* has a `latest` doc URL for 4.x. We can check with the doc team if that is planned to change.

cc @ahardin-rh

Reply:
@spadgett Correct. I don't think there is any plan to change that. 
cc @vikram-redhat